README.md

```diff
@@ -114,6 +123,10 @@ This section contains configuration fields which are used to configure the conne
 Parameter to specify the host name of the Cumulocity server. This must be configured prior to
 running the connector.
 
+#### Port (Port)
+
+Optional parameter to specify the port of the Cumulocity server.
```

Comment:
Could you please explicitly state that this parameter should be JSON string, and provide an example

Reply:
Updated to say "string parameter" and include the default value of "8883"